Dockerfile

```diff
@@ -1,6 +1,6 @@
 # MIT License
 #
-# (C) Copyright 2020-2022 Hewlett Packard Enterprise Development LP
```

Comment:
Should that be "2020-2022,2024"?

Reply:
I have always just changed the final year value so 2020-2024 should be good.